tests/snuba/api/endpoints/test_organization_events_spans_performance.py

```diff
@@ -363,6 +364,125 @@ def test_no_projects(self):
             response = self.client.get(url, format="json")
         assert response.status_code == 404, response.content
 
+    def test_bad_per_suspect(self):
+        for per_suspect in [-1000, 1000]:
```

Comment:
Perhaps bounds closer to real limits would be better? -20, 20? Super minor nit.

Reply:
used a tighter bound of -1 and 11